packages/opal-client/opal_client/data/updater.py

```diff
@@ -394,9 +394,11 @@ async def update_policy_data(
                     )
                     try:
                         await store_transaction.set_policy_data(
-                            policy_data, path=policy_store_path
+                            policy_data,
+                            path=policy_store_path,
+                            save_method=entry.save_method,
```

Comment:
I saw that the entry.save_method is type 'str', but in the description it says it should be either PUT or PATCH.
I thinks it is best to use some enum or Literal["PUT","PATCH"] to ensure that the save_method attribute has the correct value.